### PR TITLE
Update CONDA_VERSION to latest

### LINKED ISSUE #68: Outdated CONDA_VERSION causes PRs to fail
PR #67 fails in a way that's unrelated to the content of the change.

I would expect based on the [Travis log](https://travis-ci.org/rtfd/readthedocs-docker-images/builds/391868942?utm_source=github_status&utm_medium=notification) that any other PR will also fail, and that re-building the master branch will also fail.

The cause is that the Dockerfile specifies `ENV CONDA_VERSION 4.4.10`, but the latest Miniconda version as shown on <https://repo.continuum.io/miniconda/> is 4.5.4. This causes `assert current.headers['etag'] == latest.headers['etag']` in `test_dependencies.py` to fail.

### PR DESCRIPTION
**Pull request by osteele**
Fixes #68. This should clear the way for Travis to test PRs.

### CODE REVIEW

**Comment by osteele** [comment]
Currently failing because of #68 on master.

File: Dockerfile [hunk 1]
```diff
@@ -77,9 +77,9 @@ RUN pyenv install $PYTHON_VERSION_27 && \
     pyenv install $PYTHON_VERSION_36 && \
     pyenv install $PYTHON_VERSION_35 && \
     pyenv global \
-        $PYTHON_VERSION_27 \
-        $PYTHON_VERSION_36 \
-        $PYTHON_VERSION_35
+    $PYTHON_VERSION_27 \
+    $PYTHON_VERSION_36 \
+    $PYTHON_VERSION_35
```
Comment (agjohnson) [inline Dockerfile]: Can you revert this block? The spacing is intentional here.

**Comment by agjohnson** [comment]
Oh, also! We use a branched release structure here. If you want, feel free to open up a corresponding PR for our 3.0 release branch. This is only going to affect the next release of our image.

**Comment by humitos** [comment]
I'm closing this PR in favor of #74 

Thanks you all for the help here!

**Pull request closed without merging**